app/controllers/geo/GeoInformator.java

```diff
@@ -11,6 +11,11 @@
 import play.mvc.Controller;
 import play.mvc.Result;
 
+/**
+ * Methods to trigger geo lookups on the index
+ *
+ */
+@SuppressWarnings("javadoc")
```

Comment:
Ignoring missing Javadocs is fine for test classes, but for production code the reason for the custom compiler check is to actually ensure all public API is documented, so we should add documentation here.